swagger.yml

```diff
@@ -515,7 +515,7 @@ definitions:
       image:
         x-nullable: false
         type: string
-        default: coreos-stable-amd64
+        default: flatcar-stable-amd64
```

Comment:
This change effectively makes all new clusters use flatcar. Can we extract it in an extra PR for visiblity?